src/db/migrations/20181004224627-createSessions.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
```

Comment:
Is this file same as previous?

Reply:
And this migration defines the Sessions table + columns.